### PR TITLE
Patterns pagination

### PR DESCRIPTION
**Pull request by bandvov**

### CODE REVIEW

**Comment by Alan18081** [comment]
Please, resolve conflicts and add tests for patterns pagination in `pattern.query.test.js`

File: __tests__/pattern/pattern.query.test.js [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+  wrongSkip,
+  languageTypeName,
+  imageTypeName,
+} = require('./pattern.variables');
+const { PATTERN_NOT_FOUND } = require('../../error-messages/pattern.messages');
+
+let patternId;
+
+const testValue = 't e s t new';
+const patternToAdd = {
```
Comment (Alan18081) [inline __tests__/pattern/pattern.query.test.js:19]: Why does this code duplicate the code inside the `pattern.variables.js`?

File: __tests__/pattern/pattern.query.test.js [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+  wrongLimit,
+  wrongSkip,
+  languageTypeName,
+  imageTypeName,
+} = require('./pattern.variables');
+const { PATTERN_NOT_FOUND } = require('../../error-messages/pattern.messages');
+
+let patternId;
+
+const testValue = 't e s t new';
```
Comment (Alan18081) [inline __tests__/pattern/pattern.query.test.js:18]: Why does this code duplicate the code inside the `pattern.variables.js`?

File: __tests__/pattern/pattern.query.test.js [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+    medium: 'medium_335nr4j5dkebkw5cy_test.jpg',
+    small: 'small_335nr4j5dkebkw5cy_test.jpg',
+    thumbnail: 'thumbnail_335nr4j5dkebkw5cy_test.jpg',
+  },
+  material: 'test',
+  handmade: false,
+  available: true,
+};
+
+beforeAll(async () => {
```
Comment (Alan18081) [inline __tests__/pattern/pattern.query.test.js:51]: This statement should be inside the `describe` statement

File: __tests__/pattern/pattern.query.test.js [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+        }
+      `,
+
+      variables: { pattern: patternToAdd },
+    })
+    .catch(e => e);
+
+  patternId = res.data.addPattern._id;
+});
+afterAll(async () => {
```
Comment (Alan18081) [inline __tests__/pattern/pattern.query.test.js:74]: This statement should be inside the `describe` statement

File: __tests__/pattern/pattern.query.test.js [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+        available: true,
+      });
+    });
+    test('#2 Should receive one pattern', async () => {
+      try {
+        const res = await client
+          .query({
+            query: gql`
+              query($id: ID!) {
+                getPatternId(id: $id) {
```
Comment (Alan18081) [inline __tests__/pattern/pattern.query.test.js:149]: Please, rename the mutation to the `getPattern`

File: __tests__/pattern/pattern.query.test.js (hunk 1 above)
Comment (bandvov) [reply __tests__/pattern/pattern.query.test.js:19]: thats different testValue for query and mutations

File: __tests__/pattern/pattern.query.test.js (hunk 3 above)
Comment (bandvov) [reply __tests__/pattern/pattern.query.test.js:51]: fixed

File: __tests__/pattern/pattern.query.test.js (hunk 2 above)
Comment (bandvov) [reply __tests__/pattern/pattern.query.test.js:18]: thats different testValue for query and mutations

File: __tests__/pattern/pattern.query.test.js (hunk 4 above)
Comment (bandvov) [reply __tests__/pattern/pattern.query.test.js:74]: fixed

File: __tests__/pattern/pattern.query.test.js (hunk 5 above)
Comment (bandvov) [reply __tests__/pattern/pattern.query.test.js:149]: actually it was merged not correct, resolver name is getPatternById. similarly like  other resolvers

**Pull request merged**